nbgrader/docs/source/configuration/jupyterhub_config.rst

```diff
@@ -195,3 +200,135 @@ if students may be enrolled in multiple courses, you need to provide them a
 custom ``nbgrader_config.py`` that will cause nbgrader to look for assignments
 in a subdirectory corresponding to the course name. See :ref:`multiple-classes`
 for details.
+
+
+
+JupyterHub Authentication
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+
+With the advent of JupyterHubAuthPlugin students who don't have a specified course_id won't see all courses anymore, just the ones they have been added to. 
+
+Requirements for using ``JupyterHubAuthPlugin``: 
+
+* Jupyterhub > 0.8
+* Activating the JupyterHubAuthPlugin requires you to add it as an authentication plugin class into ``/etc/jupyter/nbgrader_config.py``
```

Comment:
Maybe say this instead:

```suggestion
* Activating the JupyterHubAuthPlugin requires you to add it as an authentication plugin class into the ``nbgrader_config.py`` for both the formgrader services and instructor accounts.
```

because other people may have their `nbgrader_config.py` files set up differently and paths are different on different systems. However I do want to create a separate piece of documentation (in a separate PR probably) about how to deal with different `nbgrader_config.py` files.